test/snippet/alphabet/views/trim_quality_phred42.cpp

```diff
@@ -12,14 +13,18 @@ int main()
     std::vector<seqan3::phred42> vec{"II?5+"_phred42};
 
     // trim by phred_value
-    auto v1 = vec | seqan3::views::trim_quality(20u);                            // == ['I','I','?','5']
+    auto v1 = vec | seqan3::views::trim_quality(20u);
+    seqan3::debug_stream << v1 << '\n'; // II?5
```

Comment:
why do the snippet tests fail now?

Reply:
No real reason, apparently the changes (especially in views::take) allow it to catch the error.
The variables were already unused before, but the compiler missed that.